signal_op/extract_events.m

```diff
@@ -0,0 +1,174 @@
+function event_times = extract_events(trace,varargin)
+% event_times = extract_events(trace,varargin)
+%
+% Extract event times from the ICA trace of a cell doing 2 iterations
```

Comment:
"2 iterations" of what? This comment does not explain the mathematical approach underlying this event detection script.